OpenMC_PhotonTransport.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Aug  7 10:07:33 2024
+
+@author: Anupama Rajendra
+"""
+
+import openmc
+import argparse
+import numpy as np
+
+Mesh_File = 'OpenMC_Mesh.h5m'
+Strengths = []
+#Performing simulation for the first decay time
+with open('Sum_1.txt', 'r') as Sum_1:
+    Lines = Sum_1.readlines()
+    Split = Lines[0].split()
+    for sums in Split:
+        Strengths.append(float(sums))
+
+parser = argparse.ArgumentParser(description="Specify required inputs: file path to ALARA Element Library, element name, inner radius [cm], outer_radius [cm]")
+
+#Required user inputs:
+parser.add_argument('--filepath', type=str, required=True)    
+parser.add_argument('--element_1', type=str, required=True)
+parser.add_argument('--element_2', type=str, required=True)
+#Shell radii are left as user inputs, but the mesh is specific to W_inner_radius = 1000, W_outer_radius = 1005, C_inner_radius = 995
+parser.add_argument('--W_inner_radius', type=float, required=True)
+parser.add_argument('--W_outer_radius', type=float, required=True)
+parser.add_argument('--C_inner_radius', type=float, required=True)
+
+args = parser.parse_args()
+
+fp = args.filepath
+E_1 = args.element_1
+E_2 = args.element_2
+R_W_1 = args.W_inner_radius
+R_W_2 = args.W_outer_radius
+R_C_1 = args.C_inner_radius
+
+# fp = '../../ALARA/data/elelib.std'
+# E_1 = 'W'
+# E_2 = 'C'
+# R_W_1 = 1000
+# R_W_2 = 1005
+# R_C_1 = 995
+bounds = [0, 
+                 1.00e+4, 
+                 2.00e+4, 
+                 5.00e+4, 
+                 1.00e+5, 
+                 2.00e+5, 
+                 3.00e+5, 
+                 4.00e+5, 
+                 6.00e+5, 
+                 8.00e+5, 
+                 1.00e+6, 
+                 1.22e+6, 
+                 1.44e+6, 
+                 1.66e+6, 
+                 2.00e+6, 
+                 2.50e+6, 
+                 3.00e+6, 
+                 4.00e+6, 
+                 5.00e+6, 
+                 6.50e+6, 
+                 8.00e+6, 
+                 1.00e+7, 
+                 1.20e+7, 
+                 1.40e+7, 
+                 2.00e+7]
+
+#ALARA Element Library to read density for specified element:
+# with open(""+fp+"") as ALARA_Lib:
+with open("elelib.std") as ALARA_Lib:
+    Lib_Lines = ALARA_Lib.readlines()
+
+# Create materials & export to XML:
+#Simulating tungsten shell:
+  
+def make_W(element_1):

```

Comment:
Move the geometry & materials functions into a separate python file that you can import into both neutron & photon transport